crates/bevy_render/src/camera/camera.rs

```diff
@@ -368,6 +370,24 @@ impl CameraRenderGraph {
     }
 }
 
+/// Stores Texture Views used as render targets.
+#[derive(Default, Clone, Resource, ExtractResource)]
+pub struct ManualTextureViews(HashMap<u32, (TextureView, UVec2)>);
```

Comment:
I'm not sure what "Manual" is supposed to mean here.

I think I'd prefer using a wrapper type for the u32. It wasn't immediately obvious to me what the u32 was supposed to be.

Reply:
I'd also add a note in the comment that the UVec2 is the size. Or maybe just say that it stores the texture view and it's size

Reply:
Ok great I've updated the comment.

`Manual` is intended to describe that these are texture views we are managing ourselves outside of Bevy, either by creating them in wgpu or being given them by OpenXR, WebXR etc. I agree its not obvious, any suggestions?

Reply:
I've added the `ManualTextureViewHandle` wrapper type, im a little concerned its bloating the `camera.rs` file, would you like the `ManualTextureView` struct moved to a seperate file?

Reply:
I've created `manual_texture_view.rs`, let me know if you want that reverted.